python/dgl/model_zoo/chem/README.md

```diff
@@ -0,0 +1,39 @@
+# DGL for Drug Discovery
```

Comment:
As you are now naming staff as `chem`, maybe you should change the title to be `DGL for Drug Discovery`.